MAUI/Scheduler/drag-and-drop-appointments.md

```diff
@@ -0,0 +1,191 @@
+`DragTime` - Gets the dragging time of the appointment object being dragged.
+`DragResource` - Gets the original resource of the appointment that is being dragged.
+
+{% tabs %}
+{% highlight c# %}
+scheduler.AppointmentDragOver += OnSchedulerAppointmentDragOver;
+
+...
+
+private void OnSchedulerAppointmentDragOver(object? sender, AppointmentDragOverEventArgs e)
+{
+    var appointment = e.Appointment;
+    var draggingPoint = e.DragPoint;
+    var draggingTime = e.DragTime;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Handle appointment dropping
+Using `AppointmentDrop` event you can get the dropping appointment details, position, time and you can handle whether the appointment can be dropped to the specific position or not. This event will trigger after dropping the appointment. The `AppointmentDropEventArgs` argument contains the following properties
+
+`Appointment` - Gets the appointment that is being dragged and dropped.
+`DropTime`- Gets or sets the date and time at which the appointment is being dropped.
+`SourceResource` - Gets the original resource of the appointment that is being dropped.
+`TargetResource` - Gets the resource into which the appointment is being dropped.
+
+{% tabs %}
+{% highlight c# %}
+scheduler.AppointmentDrop += OnSchedulerAppointmentDrop;
+
+...
+
+private void OnSchedulerAppointmentDrop(object? sender, AppointmentDropEventArgs e)
+{
+        var appointment = e.Appointment;
+        e.Cancel = false;
+        var dropTime = e.DropTime;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing the Drag and Drop environment
+You can handle the behavior of drag and drop in Scheduler by using `DragDropSettings` property.
+
+{% tabs %}
+{% highlight xaml %}
+<schedule:SfScheduler x:Name="dragAndDrop" ShowBusyIndicator="True"
+                    AppointmentsSource="{Binding DragEvents}"
+                    View="Week"
+                    DisplayDate="{Binding DisplayDate}"
+                    AllowedViews="Day,Week,WorkWeek,Month">
+        <schedule:SfScheduler.DragDropSettings>
+            <schedule:DragDropSettings AllowNavigation="true" AllowScroll="true" 
+                                    ShowTimeIndicator="true"/>
+        </schedule:SfScheduler.DragDropSettings>
+</schedule:SfScheduler>
+{% endhighlight %}
+{% highlight c# %}
+scheduler.DragDropSettings = new DragDropSettings 
+{
+    AllowNavigation = true,
+    AllowScroll = true,
+    ShowTimeIndicator = true,
+};
+{% endhighlight %}
+{% endtabs %}
+
+#### Disabling navigation when dragging appointment
+Using `AllowNavigate` property of `DragDropSettings` you can handle whether you can navigate views while dragging the appointment or not. The default value of the AllowNavigate property is `true`.
+
+{% tabs %}
+{% highlight c# %}
+scheduler.DragDropSettings.AllowNavigation = true;
+{% endhighlight %}
+{% endtabs %}
+
+#### Handling navigation delay while holding dragged appointment
```

Comment:
#### Handle navigation delay for dragged appointment

Reply:
changed title